### DISCUSSION TITLE
Linux install production configuration

### DISCUSSION THREAD

**Discussion by rgh256** [bionic-gpt/bionic-gpt#700] [Q&A]
Q1: Can I use the free version internally on my own servers or do I need to purchase a license?

Q2: How can I customize/supply the various passwords (e.g. database, application, readonly, smtp, etc.) when installing on my linux system in production mode (not testing/dev)? For example, is there a prototype for a config file that can be used with the `$ bionic install` command?

**Comment by tiswas247** [comment]
Sorry for the late reply. 

Q1. Yes. Licences are only for businesses etc.
Q2. The installer creates all of the secrets during installation. This ensures all secrets are large and random and that the software is secure out of the box.

**Reply by rgh256** [reply]
Thank you for clarifying! Also, thank you for open sourcing the code! I am new to Rust and devops so learning a lot from it.
